actions/frost-days/DESCRIPTION.md

```diff
@@ -0,0 +1,75 @@
+# Add metadata to fdETCCDI datafiles
+
+## Purpose
+The [p2a](https://github.com/pacificclimate/p2a-rule-engine) rule engine required monthy fdETCCDI data in order to produce `number of frost free days`.  The following commands were run in order to add metadata to the files.
+
+## Commands
+
+### 1. Add metadata common to all files
+
+Using the `update_metadata` script from [climate-explorer-data-prep](https://github.com/pacificclimate/climate-explorer-data-prep).
+
+```
+(venv)$ for file in fd/*.nc ; do python scripts/update_metadata -u all_shared.yaml $file ; done
```

Comment:
I believe that if you install `climate-explorer-data-prep` properly, the scripts are also installed and you can invoke them simply with their names; e.g., `update_metadata` instead of `python scripts/update_metadata`.

Reply:
Is that something that is done by `setup.py`?